app/views/orders/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<% if @orders.count == 0 %>
+  <h2> You have no previous orders </h2>
+  <%= link_to "Continue Shopping", items_path, class: "btn btn-warning col-sm-2" %>
+<% else %>
+
+<div class="container">
+  <div class="col-lg-12">
+    <h1 class="page-header">Order History</h1>
+  </div>
+  <table class="table table-hover table-condensed">
+    <thead>
+      <tr>
+        <th class="col-sm-6"><%= "View Order" %></th>
+        <th class="col-sm-2"><%= "Order Number" %></th>
+        <th class="col-sm-2 text-center"><%= "Status" %></th>
+        <th class="col-sm-2 text-center"><%= "Order Date" %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @orders.each do |order| %>
+      <tr>
+        <td data-th="Product">
+          <div class="row">
+            <div class="col-sm-6 text-centered">
+              <%= link_to "View Order", order_path(order.id), class: "btn btn-primary btn-large btn-block" %>
+            </div>
+          </div>
+        </td>
+        <td data-th"Order ID"><h3><%= order.id %></h3>
+          <td data-th"Status" class="text-center"><h3><%= order.status %></h3></td>
+          <td data-th="Created_at" class="text-center"><h3><%= order.created_at.strftime("%d %b. %Y") %></h3></td>
```

Comment:
This block looks similar to the block in `admin/orders/index.html.erb`. Could pull into a partial to DRY up the code.